src/microbe_stage/Microbe.Interior.cs

```diff
@@ -1092,6 +1092,8 @@ private void SpawnEjectedCompound(Compound compound, float amount)
     /// </summary>
     private Vector3 CalculateNearbyWorldPosition()
     {
+        // OLD CODE
```

Comment:
This should explain why this is kept here. Also comparing the old logic to the new, the only bit that really is different is using Membrane.GetVectorTowardsNearestPointOfMembrane, besides that the old code could be rewritten to use your approach to the vector calculation, just my using a different distance.

Reply:
I decided to use `EncompassingCircleRadius` as the other I think might be a bit computationally expensive to be run each frame.

Reply:
That's related to the new code, right? Otherwise I don't understand what you are getting at.
Finding a membrane point for venting compounds is not that performance intensive, well at least I hope, because the externally positioned organelles *each* basically call that each frame due to the position problems they run into otherwise...

Reply:
> That's related to the new code, right?

Yes. Well I think it's good to save a bit of performance with this.

Added little explanation as requested.